cpp/src/include/SingleMessage.h

```diff
@@ -271,13 +275,11 @@ namespace MiraiCP {
         * @detail 图片miraiCode格式例子, `[mirai:image:{图片id}.jpg]`
         * 可以用这个正则表达式找出id `\\[mirai:image:(.*?)\\]`
         */
-        explicit Image(std::string imageId, size_t size = 0, int width = 0, int height = 0, int type = 5)
-            : SingleMessage(Image::type(), std::move(imageId)) {
-            this->id = content; // todo(Antares): 实际上重复的属性
-            this->size = size;
-            this->width = width;
-            this->height = height;
-            this->imageType = type;
+        explicit Image(std::string imageId, size_t size = 0, int width = 0, int height = 0, int type = 5,
+                       bool isEmoji = false)
+                : SingleMessage(Image::type(), std::move(imageId)), id(imageId), size(size), width(width),
```

Comment:
```suggestion
                : SingleMessage(Image::type(), std::move(imageId)), id(imageId), size(size), width(width),
```
imageId used after moved